src/psyclone/psyir/nodes/omp_directives.py

```diff
@@ -1152,7 +1231,17 @@ def _validate_child(position, child):
         return False
 
     def gen_code(self, parent):
+        '''
+        Generate the f2pygen AST entries in the Schedule for this OpenMP
+        directive.
+        Note gen_code ignores the collapse clause but the generated code is
+        still valid.
 
+        :param parent: the parent Node in the Schedule to which to add our \
+                       content.
+        :type parent: sub-class of :py:class:`psyclone.f2pygen.BaseGen`
+
+        '''
         # We're not doing nested parallelism so make sure that this
         # omp parallel do is not already within some parallel region
         from psyclone.psyGen import zero_reduction_variables
```

Comment:
Since we're here, please could you disable the pylint warning for this import?

Reply:
Done